science-containers/Dockerfiles/carta/stable/src/nsswitch.conf

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
Since we switched from sssd/ldap to posix-mapper, the nsswitch.conf file with this config is no longer required in canfar containers.